src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse.cs

```diff
@@ -110,11 +276,31 @@ public static class Sse
         /// </summary>
         public static Vector128<float> Max(Vector128<float> left, Vector128<float> right) => Max(left, right);
 
+        /// <summary>
+        /// __m128 _mm_max_ss (__m128 a,  __m128 b)
+        /// </summary>
+        public static Vector128<float> MaxScalar(Vector128<float> left, Vector128<float> right) => MaxScalar(left, right);
```

Comment:
@fiigii, `MaxScalar` is here, `MinScalar` is a few lines up

Reply:
Thank you, Github was hiding them.